workshops/build-a-binderhub/workshop-presentations/zero-to-binderhub.md

```diff
@@ -7,13 +7,27 @@ Sarah Gibson, _The Alan Turing Institute_
 
 These steps will walk you through deploying a BinderHub on Microsoft Azure.
 It will be publicly available like [mybinder.org](https://mybinder.org).
-To follow along with these instructions, go to this link: [**bit.ly/sg-zero-to-binderhub**](http://bit.ly/sg-zero-to-binderhub)
+To follow along with these instructions, go to this link: [**bit.ly/zero-to-binderhub**](http://bit.ly/zero-to-binderhub)
```

Comment:
I get a 404 on the bitly link ☹️

Reply:
You can't redirect bitlinks on a free subscription 😢 Should work now! 🤞